pkg/nsm/apis/nseconnect/nseconnect.proto

```diff
@@ -18,8 +18,24 @@ import "github.com/ligato/networkservicemesh/pkg/nsm/apis/netmesh/netmesh.proto"
         string linux_namespace = 3;
     }
 
+// EndpointConnectionInterface is sent by a NSM to NSE to inform NSE about
+// the interface name which was created by the dataplane controller.
+    message EndpointConnectionInterface {
+        string request_id = 1;
+        string interface_name = 2;
```

Comment:
What if the LocalMechanism is not an interface?

Reply:
Examples please

Reply:
Memif

Reply:
it is strange, from vpp perspective memif is interface with unique id. There is no name, true, but there is a combination of a socket + id which memif client will need to know.

Reply:
@sbezverk What I was thinking was roughly like this:

1.  We could simply use the fully specified LocalMechanism.  That will have both the type selected and all the necessary parameters for each type.
2.  We have the option of getting this down to a single call to RequestEndpointConnection.  The NSM has enough information to program the cross connect before the call to RequestEndpointConnection... in which case it can simply pass the fully specified LocalMechanism in that RequestEndpointConnection call.